tests/platform_tests/api/test_thermal.py

```diff
@@ -111,45 +135,49 @@ def test_get_temperature(self, duthost, localhost, platform_api_conn):
+                    self.compare_value_with_platform_facts('low_threshold', low_threshold, i)
         self.assert_expectations()
 
     def test_get_high_threshold(self, duthost, localhost, platform_api_conn):
         # Ensure the thermal high threshold temperature is sane
         for i in range(self.num_thermals):
-            temperature = thermal.get_high_threshold(platform_api_conn, i)
+            high_threshold = thermal.get_high_threshold(platform_api_conn, i)
 
-            if self.expect(temperature is not None, "Unable to retrieve Thermal {} high threshold".format(i)):
-                if self.expect(isinstance(temperature, float), "Thermal {} high threshold appears incorrect".format(i)):
-                    self.expect(temperature > 0 and temperature <= 100,
-                                "Thermal {} high threshold {} reading is not within range".format(i, temperature))
+            if self.expect(high_threshold is not None, "Unable to retrieve Thermal {} high threshold".format(i)):
+                if self.expect(isinstance(high_threshold, float), "Thermal {} high threshold appears incorrect".format(i)):
+                    self.expect(high_threshold > 0 and high_threshold <= 100,
+                                "Thermal {} high threshold {} reading is not within range".format(i, high_threshold))
+                    self.compare_value_with_platform_facts('high_threshold', high_threshold, i)
         self.assert_expectations()
 
     def test_get_low_critical_threshold(self, duthost, localhost, platform_api_conn):
         # Ensure the thermal low critical threshold temperature is sane
         for i in range(self.num_thermals):
-            temperature = thermal.get_low_critical_threshold(platform_api_conn, i)
+            low_critical_threshold = thermal.get_low_critical_threshold(platform_api_conn, i)
 
-            if self.expect(temperature is not None, "Unable to retrieve Thermal {} low critical threshold".format(i)):
-                if self.expect(isinstance(temperature, float), "Thermal {} low threshold appears incorrect".format(i)):
-                    self.expect(temperature > 0 and temperature <= 110,
-                                "Thermal {} low critical threshold {} reading is not within range".format(i, temperature))
+            if self.expect(low_critical_threshold is not None, "Unable to retrieve Thermal {} low critical threshold".format(i)):
+                if self.expect(isinstance(low_critical_threshold, float), "Thermal {} low threshold appears incorrect".format(i)):
+                    self.expect(low_critical_threshold > 0 and low_critical_threshold <= 110,
+                                "Thermal {} low critical threshold {} reading is not within range".format(i, low_critical_threshold))
+                    self.compare_value_with_platform_facts('low_critical_threshold', low_critical_threshold, i)
```

Comment:
For now we are not storing threshold values in platform.json. Please remove this check.

Reply:
fixed